### PR TITLE
Feature/testimonials

### PR DESCRIPTION
**Pull request by Poison159**
Added testimonials section to the Home Page

Screenshot 2022-03-30 at 14 57 22

### CODE REVIEW

**Comment by Zandile** [comment]
@Poison159 happy with this. I will ask @mukwevhom to merge

File: src/components/Testimonials.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+const cardImg = {
+    borderRadius:"100%",
+    width:"40%",
+    height:"40%",
+    marginLeft:"auto",
+    marginRight:"auto"
+}
+
+const Testimaonials = () => {
```
Comment (mukwevhom) [inline src/components/Testimonials.js:11]: Please check the spelling here

File: src/components/Testimonials.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    return (
+        <div style={{textAlign:"center"}} className="row">
```
Comment (mukwevhom) [inline src/components/Testimonials.js:13]: I'd suggest that you either use a bootstrap class or move it to the styles file

File: src/components/Testimonials.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            <h4>OUR GIRL CODERS</h4>
```
Comment (mukwevhom) [inline src/components/Testimonials.js:14]: Rather we use styles to make this uppercase

**Review by mukwevhom — CHANGES_REQUESTED** [review]
@Poison159 Please check some of the suggestions I have added here

File: src/components/Testimonials.js (hunk 3 above)
Comment (Poison159) [reply src/components/Testimonials.js]: Hey there
I have made the requested changes and committed them to this branch -> [https://github.com/GirlCodeZA/girlcode-website-project/pull/10/commits/bed3550b3b56a704405392ffe21154b838aaa864](url)

Please let me know if there is anything else you would like me to do.

Comment (Poison159) [reply src/components/Testimonials.js]: Hey @mukwevhom i have moved the css to a css file. 

Please let me know if you need anything else

Comment (Poison159) [reply src/components/Testimonials.js]: oh here is where it is -> https://github.com/GirlCodeZA/girlcode-website-project/pull/10/commits/d572f556fecb27a5db1635957ea848db01627800

**Review by mukwevhom — APPROVED** [review]
Looks good to me

**Pull request merged**